main.go

```diff
@@ -61,25 +61,27 @@ var rootCmd = &cobra.Command{
 }
 
 var (
-	port                int
-	tilePath            string
-	certificate         string
-	privateKey          string
-	rootURLStr          string
-	domain              string
-	secretKey           string
-	sentryDSN           string
-	verbose             bool
-	autotls             bool
-	redirect            bool
-	enableReloadSignal  bool
-	enableReloadFSWatch bool
-	generateIDs         bool
-	enableArcGIS        bool
-	disablePreview      bool
-	disableTileJSON     bool
-	disableServiceList  bool
-	tilesOnly           bool
+	port                 int
+	tilePath             string
+	certificate          string
+	privateKey           string
+	rootURLStr           string
+	domain               string
+	secretKey            string
+	sentryDSN            string
+	verbose              bool
+	autotls              bool
+	redirect             bool
+	enableReloadSignal   bool
+	enableReloadFSWatch  bool
+	enableReloadEndpoint bool
```

Comment:
Food for thought: why not merging `enableReloadEndpoint` and `reloadToken` into `enableReloadToken`? As one requires the other, the result is simpler and less error-prone.

Reply:
good idea! working on this change